testem.json

```diff
@@ -1,11 +1,21 @@
 {
   "framework": "mocha",
-  "test_page": "tests/index.html?hidepassed&coverage",
+  "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
   "launch_in_ci": [
-	"Firefox"
+    "Firefox"
   ],
   "launch_in_dev": [
-	"Chrome"
-  ]
+    "Firefox"
```

Comment:
I understand that being able to have Firefox run in dev since it is what is running in CI is valuable but I also think we should have Chrome in here as well. Then we have both Firefox and Chrome being covered.